maps/away/mining/mining.dm

```diff
@@ -40,9 +40,12 @@
 		/area/mine/explored = NO_SCRUBBER|NO_VENT|NO_APC,
 		/area/mine/unexplored = NO_SCRUBBER|NO_VENT|NO_APC
 	)
-	area_usage_test_exempted_root_areas = list(/area/mine)
+	area_usage_test_exempted_root_areas = list(/area/mine, /area/spacestations)
 	area_usage_test_exempted_areas = list(/area/djstation)
 	area_coherency_test_exempt_areas =  list(/area/mine/explored, /area/mine/unexplored)
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
```

Comment:
if we're adding a shuttle between the trading station and this away, you should readd the guaranteed spawn location near the NT station so you're not teleporting across the sector. Should also set the budget cost for this away to 0.